### ISSUE TITLE
ModuleNotFoundError: No module named 'pyteamup.calendar'

### ISSUE THREAD

**Issue by vranki** [Djones4822/pyTeamUp#2]
Hello,

I'm trying to use the library in a docker container and for some reason loading the library fails:

```
>>> from pyteamup import Calendar, Event
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/pyteamup/__init__.py", line 14, in <module>
    from pyteamup.calendar import Calendar
ModuleNotFoundError: No module named 'pyteamup.calendar'

cat /usr/local/lib/python3.7/site-packages/pyteamup/__init__.py
"""

pyteamup

Python Library for interfacing with the TeamUp api

"""

__version__ = '0.0.1a'
__author__ = 'David Jones'

name = 'pyTeamUp'

from pyteamup.calendar import Calendar
from pyteamup.event import Event


ls /usr/local/lib/python3.7/site-packages/pyteamup/
Calendar.py  __init__.py  __pycache__  event.py  subcalendar.py  utils


# python -m pip install pyTeamUp
Requirement already satisfied: pyTeamUp in /usr/local/lib/python3.7/site-packages (0.1.0a2)
```
Is there something wrong with the pip packaging?

Version number in __init__.py seems to be wrong at least.

**Comment by vranki** [comment]
Ok got it imported with these hacks:

* Needs to install numpy and panda (should these be dependencies?)
* /usr/local/lib/python3.7/site-packages/pyteamup/__init__.py imports must be changed:
from pyteamup import Calendar
from pyteamup import event
* Import with from pyteamup import Calendar, event (event with lowercase e)

**Comment by Djones4822** [comment]
Very sorry about this. Resolved all imports, fixed naming scheme, fixed the versioning, and fixed setup.py to include the requirements appropriately. Numpy is required, but pandas is not. Pandas is only used if you have it, but numpy is needed for the to_datetime function  

Let me know if you have any problems, I tested it in a virtual environment on my machine and it worked. Hopefully no issue on docker.
